docs/edit_first_game_server.md

```diff
@@ -33,20 +37,31 @@ go get agones.dev/agones/pkg/sdk
 GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o bin/server -a -v main.go
 ```
 
-## Using Docker File without using Minikube
+## Using Docker File
 
 ### Create a new docker image
 ```bash
 docker build -t gcr.io/[PROJECT_ID]/agones-udp-server:0.2 .
 ```
 Note that: you can change the image name "agones-udp-server" to something else.
 
-### Push the image to GCP Registry
+### If using GKE, push the image to GCP Registry
 ```bash
 docker push gcr.io/[PROJECT_ID]/agones-udp-server:0.2
 ```
 
-### Modify gameserver.yaml
+### If using Minikube, change directory to agones/build and push the image into Minikube
+```bash
```

Comment:
Rather than point people to development tools, we can just show them the command line trick:
(just concerned with Agones users relying on development tooling - in case it changes).

`docker save [PROJECT_ID]/agones-udp-server:0.2 | (eval $(minikube docker-env) && docker load)`

WDYT?

Reply:
Yes that makes sense.  It seems less complex.  I tried both ways and eval $(minikube docker-env) etc... worked fine provided the make tools weren't used.

Reply:
I'm thinking we shouldn't be pointing people towards the development tooling - it's very cross purpose, and developers shouldn't be coupled to our own development tooling.

Reply:
The doc now uses docker save [PROJECT_ID]/agones-udp-server:0.2 | (eval $(minikube docker-env) && docker load) instead of make.